### ISSUE TITLE
TokenStrategy is not a function

### ISSUE THREAD

**Issue by naptiv** [feathersjs/generator-feathers#57]
```
/node_modules/feathers-authentication/lib/services/oauth2/index.js:46
      _passport2.default.use(new TokenStrategy(options, service.oauthCallback.bind(service)));
                             ^

TypeError: TokenStrategy is not a function
```
I've got this at npm run, any idea ?

**Comment by naptiv** [comment]
It was missing the config of the passport stragegy

**Comment by ekryski** [comment]
@naptiv were you just missing the `tokenStrategy`piece? We might be able to provide some better warnings or error handling.

**Comment by naptiv** [comment]
Hi @ekryski 
I guess that's about : 
- the setup in config/default.json on facebook & linkedin
- the setup un services/authentication.index.js

I've commented the stategies not yet used, I'll give a try to setup them correctly and see what happen.

**Comment by ekryski** [comment]
Fixed with release v0.6.8
